drivers/eks/templates.go

```diff
@@ -186,7 +212,7 @@ Parameters:
   NodeInstanceType:
     Description: EC2 instance type for the node instances
     Type: String
-    Default: t2.medium
+    Default: t3.medium
```

Comment:
Why are we changing the default?

Reply:
Because the [template](https://amazon-eks.s3-us-west-2.amazonaws.com/cloudformation/2019-01-09/amazon-eks-nodegroup.yaml) from aws changed the default. Line 18